Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1547,7 +1547,7 @@ NOTE: Restart is required for the setting to take effect.</value>
     <value>No code module found for the target module '{0}'.</value>
     <comment>{0}: name of module</comment>
   </data>
-  <data name="RenameFailure_TargetContriolNotFound" xml:space="preserve">
+  <data name="RenameFailure_TargetControlNotFound" xml:space="preserve">
```

Comment:
I think you should update the bad key from other localized resource files or it'll break in non-English installs.

Reply:
Thank you -> really good catch (I have to start thinking more globally when writing RD code).  Updated where the keys existed (missing in spanish and french)